.changeset/tidy-olives-dance.md

```diff
@@ -0,0 +1,7 @@
+---
+"@fuel-ts/abi-typegen": patch
+"@fuel-ts/forc": patch
+"@fuel-ts/versions": patch
+---
+
+chore: upgrading forc to `0.51.1`
```

Comment:
> [!WARNING]
> Do not release before public network compatibility.

Reply:
I moved the warning note to the PR description and marked it as a draft.

PR will be deleted soon.